datawell/iex.py

```diff
@@ -16,6 +16,7 @@ def __init__(self):
         self.Symbols = self.get_stocks()
         datapoints = ['logo', 'company']
         self.Datapoints = dict(zip(datapoints, datapoints))
+        self.companyList=[]
```

Comment:
JS haunts you, does it not? :)
Pythonic convention will be:
.PublicProperty
._private_field
.local_var